### PR TITLE
Add dupe_import rule

### PR DESCRIPTION
**Pull request by scheibo**
Check for cases where `@import` is called multiple times with the same value within a file.

### CODE REVIEW

File: src/main.zig [hunk 1]
```diff
@@ -470,6 +476,7 @@ fn lint(
                         "line is {s}{} characters long{s}; the maximum is {}",
                         .{ red_text, len, end_text_fmt, analyzer.max_line_length },
                     ),
+                    .DupeImport => |name| try stdout_writer.print("found duplicate import of {s}", .{name}),
```
Comment (AnnikaCodes) [inline src/main.zig]: We can add color highlighting here. 
```suggestion
                    .DupeImport => |name| try stdout_writer.print(
                        "found duplicate import of {s}{s}{s}",
                        .{bold_magenta, name, end_text_fmt},
                    ),
```

File: src/rules/dupe_import.zig [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    pub fn check_node(
+        self: *DupeImport,
+        allocator: std.mem.Allocator,
+        fault_tracker: *analysis.SourceCodeFaultTracker,
+        tree: std.zig.Ast,
+        node_idx: u32,
+    ) !void {
+        _ = .{ self, allocator, fault_tracker };
```
Comment (AnnikaCodes) [inline src/rules/dupe_import.zig]: What does this do?

**Review by AnnikaCodes — COMMENTED** [review]
Looking good, thanks!

File: src/rules/dupe_import.zig (hunk 2 above)
Comment (scheibo) [reply src/rules/dupe_import.zig]: this was to ignore unused params while developing and I forgot to remove

**Review by AnnikaCodes — APPROVED** [review]

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**